packages/e2e-tests/specs/change-detection.test.js

```diff
@@ -133,14 +133,14 @@ describe( 'Change detection', () => {
 		await assertIsDirty( false );
 	} );
 
-	it( 'Should not prompt to confirm unsaved changes for new post with initial edits', async () => {
+	it( 'Should prompt to confirm unsaved changes for new post with initial edits', async () => {
 		await createNewPost( {
 			title: 'My New Post',
 			content: 'My content',
 			excerpt: 'My excerpt',
 		} );
 
-		await assertIsDirty( false );
+		await assertIsDirty( true );
```

Comment:
Why has this changed?

Reply:
Initial edits should be considered a dirtying operation.

See https://github.com/WordPress/gutenberg/pull/16932#discussion_r315556523